R/predict.R

```diff
@@ -197,14 +195,27 @@ check_pred_type <- function(object, type, ...) {
         glue_collapse(pred_types, sep = ", ", last = " and ")
       )
     )
-  if (type == "numeric" & object$spec$mode != "regression")
+  if (type == "numeric" & object$spec$mode != "regression") {
```

Comment:
```suggestion
  if (type == "numeric" && object$spec$mode != "regression") {
```

This suggestion applies to all `if (...)`